backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Tests/Altinn.AuthorizationAdmin.Tests.csproj

```diff
@@ -66,5 +69,11 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Please remove all project file clutter for individual files under the Data folder, which should be unnecessary and already covered by the catch all item group.